lib/ui.js

```diff
@@ -31,7 +31,12 @@ function prettyVersionDiff(oldVersion, inc) {
 module.exports = options => {
 	const pkg = util.readPkg();
 	const oldVersion = pkg.version;
-	const repositoryUrl = githubUrlFromGit(pkg.repository.url);
+	let repositoryUrl;
+	try {
```

Comment:
After looking at it again, it would be better to explicitly check for the URL property.

```js
if (!pkg.repository || !pkg.repository.url) {
   throw new Error('...');
}
```

Reply:
Sure! Will update it and push it soon. The error text is ok? I’m terrible at this thing called “phrasing” :P

Reply:
I leave this up to @sindresorhus. Not sure about the text coloring though. I believe I would write it like

> Could not determine package repository URL from package.json

Reply:
I’ll wait for @sindresorhus to pitch in, then, and update it at once :) Thanks, Sam!